pkg/state/helmx.go

```diff
@@ -100,7 +100,10 @@ func (st *HelmState) PrepareChartify(helm helmexec.Interface, release *ReleaseSp
 
 	var shouldRun bool
 
-	dir := filepath.Join(st.basePath, chart)
+	dir := chart
+	if !filepath.IsAbs(chart) {
+		dir = filepath.Join(st.basePath, chart)
```

Comment:
Gotcha! Yeah, we should do this everywhere in Helmfile.
I thought I had already made helmfile relies on absolute paths everywhere to avoid potential errors like this one, but apparently, I missed a few or more...